components/server/src/code-sync/code-sync-service.ts

```diff
@@ -253,24 +253,10 @@ export class CodeSyncService {
                 if (response.status !== 200) {
                     throw new Error(`code sync: blob service: upload failed with ${response.status} ${response.statusText}`);
                 }
-                oldObject = oldRev && toObjectName(resourceKey, oldRev);
+                oldRevList = oldRevs
             }, { latestRev, revLimit });
-            if (oldObject) {
-                const request = new DeleteRequest();
-                request.setOwnerId(userId);
-                request.setExact(oldObject);
-
-                const blobsClient = this.blobsProvider.getDefault();
-                blobsClient.delete(request, (err: ServiceError | null) => {
-                    if (err) {
-                        if (err.code === status.NOT_FOUND) {
-                            // we're good here
-                            return;
-                        }
-                        log.error({ userId }, 'code sync: failed to delete', err, { object: oldObject });
-                    }
-                });
-            }
+            // sync delete old revs from storage
+            this.deleteObjects(userId, resourceKey, oldRevList)
```

Comment:
missing semicolon

Reply:
That's why we need some lint !!

Reply:
I create a temp .prettierrc file like this to reformat temporarily
```json
{
  "tabWidth": 4,
  "useTabs": false,
  "printWidth": 400,
  "singleQuote": true,
  "arrowParens": "avoid"
}
```

Reply:
@Harry-Hopkinson created a global lint for us 🍊 https://github.com/gitpod-io/gitpod/pull/8607 and @easyCZ created one too https://github.com/gitpod-io/gitpod/pull/8655!!

Hope them can be merged soon 👍

Reply:
This has now landed. Not all components are enabled by default but `server` is.